eng/pipelines/templates/stages/1es-redirect.yml

```
@@ -36,7 +39,15 @@ extends:
         - 1ES.PT.Tag-refs/tags/canary
     settings:
       skipBuildTagsForGitHubPullRequests: true
+       # only enable autoBaseline for the internal build of rust-core on main branch
```

Nit: one space too far
```suggestion
      # only enable autoBaseline for the internal build of rust-core on main branch
```